person.rb

```diff
@@ -1,27 +1,37 @@
 require_relative 'nameable'
 require_relative 'capitalize_decorator'
 require_relative 'trimmer_decorator'
+require_relative 'rental'
 
 class Person < Nameable
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
   attr_reader :id
 
-  def initialize(age, parent_permission: true, name: 'Unknown')
+  def initialize(age, name: 'Unknown', parent_permission: true)
     super()
     @id = generate_random_id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def correct_name
     name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Let's improve this association method to be like the one in the Book class.

  ```
  def add_rental(date, book)
    Rental.new(date, self, book)
  end
  ```